framework/validators/ExistValidator.php

```diff
@@ -73,6 +73,11 @@ class ExistValidator extends Validator
      */
     public $targetAttributeJunction = 'and';
 
+    /**
+     * @var bool whether this validator can run on salve connections
+     */
+    public $canValidateOnSlaveDb =  true;
```

Comment:
"can" is ambiguous. It means validator can run on slave but it doesn't specify it would run on slave. That's confusing.